ucm2/USB-Audio/USB-Audio.conf

```diff
@@ -145,6 +145,15 @@ If.id4 {
 	True.Define.ProfileName "Audient/Audient-iD4"
 }
 
+If.M4 {
+	Condition {
+		Type String
+		Haystack "${CardComponents}"
+		Needle "USB07fd:000b"
```

Comment:
A bit nitpicking - I'm trying to have the conditions sorted by USB ID value, so please, put `If.M4` before `If.dell-wd15`. Thanks.

Reply:
All good. Moved m4 to right before dell-wd15.